bit/prjoxide/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+set -x
+
+curl --proto '=https' -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
Are we sure it's installed within the env? Are there any switches to ensure that, maybe? I assume we do it this way not to use conda-forge?

Reply:
Hmm, actually I can try with conda-forge and see if there are compatibility issues with the only other 2 capnproto packages

Reply:
I have checked and using conda-forge generates conflicts